acct-user/systemd-coredump/systemd-coredump-0-r1.ebuild

```diff
@@ -0,0 +1,11 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+ACCT_USER_ID=194
```

Comment:
This user seems to be missing from https://github.com/flatcar-linux/baselayout/blob/flatcar-master/baselayout/passwd. I don't know if this is important, though.